File: C/Iniciante/1008.c
#include <stdio.h>
 
int main() {
 
    int number, horas;
    double salario, salarioTotal;
    
    printf("");
    scanf("%d", &number);
    printf("");
    scanf("%d", &horas);
    printf("");
    scanf("%lf", &salario);
    
    salarioTotal = (horas * salario);
    
    printf("NUMBER = %d\n", number);
    printf("SALARY = U$ %.2lf\n", salarioTotal);
 
    return 0;
}

File: Dart/Iniciante/1008.dart
import 'dart:io';

void main() {

    String? valorID = stdin.readLineSync();
    String? valorSAL = stdin.readLineSync();
    String? valorSAL2 = stdin.readLineSync();
    
    if (valorID != null && valorSAL != null && valorSAL2 != null) {
      double salario = double.parse(valorSAL);
      double horasTrabalhadas = double.parse(valorSAL2);
      double salarioFinal = salario * horasTrabalhadas;
      print('NUMBER = $valorID');
      print('SALARY = U\$ ${salarioFinal.toStringAsFixed(2)}');
    } else {
      print('Erro: Certifique-se de fornecer valores válidos.');
    }
}
